### DISCUSSION TITLE
No docker images after version 1.15.1 for aarch64

### DISCUSSION THREAD

**Discussion by smoreau** [dani-garcia/vaultwarden#1439] [Q&A] [answered]
I am currently running bitwardenrs on a Raspberry Pi.
I am using the latest version available on the Docker Hub for aarch64, which is the version 1.15.1: https://hub.docker.com/r/bitwardenrs/server/tags?page=1&ordering=last_updated&name=aarch64

Why the newest versions are not available on Docker Hub?
Is there another way to get these images?

**Comment by BlackDex** [comment, marked as answer]
Just use `latest` or `testing` we are using multi arch tagged versions.
Also, its called arm64 now instead of aarch64.

See: https://hub.docker.com/layers/bitwardenrs/server/latest/images/sha256-200534287a2b3aff7eee8eaa8be2ba87198c9f0a331ae1f3fbc15e57f6933072?context=explore

**Comment by smoreau** [comment]
Thanks a lot for your answer.
The image in arm64 works perfectly fine.
